playwright/helpers/auth.ts

```
@@ -8,10 +8,7 @@
  * authentication. These functions are for special cases that need manual login.
  */
 import type { Page } from '@playwright/test';
-import {
-  login as sharedLogin,
-  disableCookiePrompt
-} from '@redhat-cloud-services/playwright-test-auth';
+import { login as sharedLogin, disableCookiePrompt } from '@redhat-cloud-services/playwright-test-auth';
```

ESLint

[error] Member 'disableCookiePrompt' of the import declaration should be sorted alphabetically.

(sort-imports)